CONTRIBUTING.md

```diff
@@ -1,94 +1,121 @@
 Contributing
 ============
 
-In this document: 
+Thank you for your interest in contributing to the .NET documentation!
+
+In this topic, you'll see the basic process for adding or updating content in our [documentation site](https://docs.microsoft.com/dotnet). For the detailed step-by-step process and instructions, please see the [official guide](https://github.com/Microsoft/Docs/blob/master/readme.md) in the [Microsoft/Docs](https://github.com/Microsoft/Docs) repo.
+
+In this topic, we'll cover: 
 
 * [Adding content](#adding-content)
 * [Process for contributing](#process-for-contributing) 
 * [Guidance checklist](#guidance-checklist)
 * [Building the docs](#building-the-docs)
 * [Contributing to samples](#contributing-to-samples)
+* [Contributor License Agreement](#contributor-license-agreement)
 
 ## Adding content ##
```

Comment:
The end of line pound signs are unnecessary; I suggest we eliminate them unless there's some reason to keep them that I'm not aware of.